### PR TITLE
add or change k8s v1.27

### PR DESCRIPTION
**Pull request by takara9**

### CODE REVIEW

**Review by kmdkuk — COMMENTED** [review]
Please update the package version in go.mod and go version.

File: controllers/virtualdc_controller_test.go [hunk 1]
```diff
@@ -497,6 +498,7 @@ kind: Pod`
 	})
 
 	It("should recreate the service resource when the service resource is deleted", func() {
+
```
Comment (kmdkuk) [inline controllers/virtualdc_controller_test.go:501]: Is this empty line intentional?

File: go.mod [hunk 2]
```diff
@@ -1,59 +1,63 @@
...
-	k8s.io/api v0.26.4
-	k8s.io/apimachinery v0.26.4
-	k8s.io/client-go v0.26.4
-	k8s.io/utils v0.0.0-20230505201702-9f6742963106
-	sigs.k8s.io/controller-runtime v0.14.6
+	github.com/onsi/ginkgo/v2 v2.13.0
+	github.com/onsi/gomega v1.28.1
+	k8s.io/api v0.28.3
+	k8s.io/apimachinery v0.28.3
+	k8s.io/client-go v0.28.3
```
Comment (kmdkuk) [inline go.mod:11]: It is desirable to use the latest version of 0.27.Z for these three packages.

File: pkg/entrypoint/server_test.go [hunk 3]
```diff
@@ -58,72 +58,72 @@ var _ = Describe("entrypoint status API test", func() {
...
 					{
-						Name:    "test2",
+						Name:    "test8",
 						Command: "sleep",
-						Args:    []string{"1"},
+						Args:    []string{"5"},
 					}},
 				expected: []statusResponse{
-					{Jobs: []job{{Name: "test1", Status: "Failed"}, {Name: "test2", Status: "Pending"}}},
+					{Jobs: []job{{Name: "test7", Status: "Failed"}, {Name: "test8", Status: "Pending"}}},
```
Comment (kmdkuk) [inline pkg/entrypoint/server_test.go:126]: Why did you make the test names a sequential number?
I thought there would be no problem if Name was unique within each `[]Job`.

File: controllers/virtualdc_controller_test.go (hunk 1 above)
Comment (takara9) [reply controllers/virtualdc_controller_test.go:501]: Not,  I will delete this line.

File: go.mod (hunk 2 above)
Comment (takara9) [reply go.mod:11]: It had automatically updated due to depend on the version of the runtime-controller.

File: pkg/entrypoint/server_test.go (hunk 3 above)
Comment (takara9) [reply pkg/entrypoint/server_test.go:126]: Yes, it is unesessary update, but it bring us easy identify test reult when we confuse which case failed.

**Review by kmdkuk — APPROVED** [review]

**Pull request merged**